registration/views.py

```diff
@@ -825,12 +835,19 @@ def form_valid(self, form):
                 form.add_error(field=None, error=error_non_field_names)
             return self.form_invalid(form)
 
-        return super(EmailNextOfKinView, self).form_valid(form)
+        return super(EmailNextOfKinBaseView, self).form_valid(form)
 
     def submit_nok_email(self, data):
         try:
             data_copy = data.copy()
+            data_copy['variant_id'] = self.variant_id
             data_copy['from_email'] = self.request.session.get(SESSION_EMAIL, settings.DEFAULT_FROM_EMAIL)
+            body = "{}{}".format(data_copy['body'], _(
```

Comment:
Upon looking at this, I think we shouldn't handle putting the HTML in here. Its dangerous since a user could put in dangerous HTML.

Why don't we move this to 53, and in 53 we can strip all HTML from the submitted content (this is actually a finding of the security test that we need to fix anyway), convert it to HTML on the server (I think there's a built-in django function for this) and add the extra line at the end. We will know which variant is submitted along with the API call, and can add the ``variant_id`` URL variable server side in 53.